File: Contagem de números entre o primeiro dado e N.java
public class Main {
    public static int countInRange(int[] arr, int N) {
        int first = arr[0];
        int count = 0;
        for (int num : arr) {
            if (num >= first && num <= N) {
                count++;
            }
        }
        return count;
    }

    public static void main(String[] args) {
        int[] arr = {1, 2, 3, 4, 5};
        System.out.println(countInRange(arr, 4));  
    }
}

File: Contagem de números entre o primeiro dado e N.js
function countInRange(arr, N) {
    let first = arr[0];
    let count = 0;
    for (let num of arr) {
        if (num >= first && num <= N) {
            count++;
        }
    }
    return count;
}

console.log(countInRange([1, 2, 3, 4, 5], 4));  
